API_CHANGES.md

```diff
@@ -22,13 +42,26 @@
   * This field type is `Either<AssistSonarQubeConnection, AssistSonarCloudConnection>`
   * Common methods of both connection types are added to the `AssistCreatingConnectionParams` class to provide users simplicity
 
+### File events
+
+* Add the `didOpenFile` and `didCloseFile` methods to `org.sonarsource.sonarlint.core.rpc.protocol.backend.file.FileRpcService`.
+  * Clients are supposed to call these methods when a file is opened in the editor or closed.
+
+### Analysis
+
+* Add a new constructor in `org.sonarsource.sonarlint.core.rpc.protocol.backend.initialize.InitializeParams` to let clients provide if automatic analysis is enabled.
+* Add a new `didChangeAutomaticAnalysisSetting` method in `org.sonarsource.sonarlint.core.rpc.protocol.backend.analysis.AnalysisRpcService`
+  * Clients are expected to call it whenever users change the "enable automatic analysis" setting.
```

Comment:
We should also document the new `analyze*` methods in the analysis service

Reply:
Added.